src/homework/hom20/Builder/CarBuilderImpl.java

```diff
@@ -0,0 +1,49 @@
+package homework.hom20.Builder;
+
+public class CarBuilderImpl implements CarBuilder {
```

Comment:
It's a not builder pattern. 
You need to return this in each method inside

Reply:
corrected